pkg/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -235,7 +235,19 @@ func (q *DBJobQueue) Enqueue(jobType string, args interface{}, dependencies []uu
 	}()
 
 	id := uuid.New()
-	_, err = tx.Exec(context.Background(), sqlEnqueue, id, jobType, args, channel)
+
+	var deltaDays int64
+
+	if expiry != 0 {
+		deltaDays = expiry
+	} else {
+		// A '0' being passed means 'no expiry' (aka: very very long time)
+		deltaDays = 9999
+	}
+
+	expires := time.Now().UTC().Add(time.Hour * time.Duration(24*deltaDays))
```

Comment:
I think that the `time.Duration` cast doesn't need to be there. Correct me if I'm wrong, I find it slightly confusing.